app/views/NewUkimsNumberView.scala.html

```diff
@@ -0,0 +1,68 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.LabelSize._
+@import viewmodels.InputWidth._
+@import models.Mode
+
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], action: Call, isCreateJourney: Boolean = false)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("ukimsNumber.title"))) {
```

Comment:
can we use newUkimsNumber for all the content message tags - just because it's a brand new view now (even though most of the messages are the same)